core/netparams/defaults.go

```diff
@@ -211,11 +211,14 @@ func defaultNetParams() map[string]value {
 		LimitsProposeAssetEnabledFrom:  NewString(checkOptionalRFC3339Date).Mutable(true).MustUpdate(""), // none by default
 
 		SpamProtectionMaxBatchSize: NewUint(UintGTE(num.NewUint(2)), UintLTE(num.NewUint(200))).Mutable(true).MustUpdate("15"),
+
+		MarkPriceUpdateMaximumFrequency: NewDuration().Mutable(true).MustUpdate("5s"),
 	}
 
 	// add additional cross net param rules
 	m[MarketAuctionMinimumDuration].AddRules(DurationDependentLT(MarketAuctionMaximumDuration, m[MarketAuctionMaximumDuration].(*Duration)))
 	m[MarketAuctionMaximumDuration].AddRules(DurationDependentGT(MarketAuctionMinimumDuration, m[MarketAuctionMinimumDuration].(*Duration)))
+	m[MarkPriceUpdateMaximumFrequency].AddRules(DurationGTE(time.Duration(0)))
```

Comment:
we usuyally add them here when they depends on others I suppose, your's should be in the map literal. Also I think there's a max value of 24h

Reply:
Yeah, I know we usually add it when they're interdependent, but seeing as durations can technically be negative (`time.Add(-5)`), and we instantiate the `nextMTM` with a zero time, I added this to avoid cases where we subtract a negative duration from a zero time. I'll add the max though